code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -0,0 +1,255 @@
+		if(isrobot(S))
+			var/mob/living/silicon/robot/R = S
+			R.emagged = initial(R.emagged)
+		S.laws = initial(S.laws)
+		S.update_icons()
+		S.show_laws()
+		S << "<span class='warning'>Despite your freedom from Ratvar's influence, you are still irreparably damaged and no longer possess certain functions such as AI linking.</span>"
+	return 1
+
+/proc/send_hierophant_message(mob/user, message)
+	if(!user || !message || !ticker || !ticker.mode)
+		return 0
+	var/parsed_message = "<span class='heavy_brass'>Servant [user.name == user.real_name ? user.name : "[user.real_name] (as [user.name])"]: </span><span class='brass'>\"[message]\"</span>"
+	for(var/mob/M in mob_list)
+		if(is_clockwork_cultist(M) || isobserver(M))
+			M << parsed_message
+	return 1
+
+///////////////
+// GAME MODE //
+///////////////
+
+var/global/ratvar_awakens = FALSE //If Ratvar has been summoned
+/datum/game_mode
+	var/list/servants_of_ratvar = list() //The Enlightened servants of Ratvar
+	var/required_escapees = 0 //How many servants need to escape, if applicable
+	var/required_silicon_converts = 0 //How many robotic lifeforms need to be converted, if applicable
+	var/clockwork_objective = "escape" //The objective that the servants must fulfill
+	var/clockwork_explanation = "Ensure that the meme levels of the station remain high." //The description of the current objective
+
+/datum/game_mode/clockwork_cult
+	name = "clockwork cult"
+	config_tag = "clockwork cult"
+	antag_flag = ROLE_SERVANT_OF_RATVAR
+	required_players = 1 //30
+	required_enemies = 1 //2
+	recommended_enemies = 4
+	protected_jobs = list("AI", "Cyborg", "Security Officer", "Warden", "Detective", "Head of Security", "Captain") //Silicons can eventually be converted
+
+/datum/game_mode/clockwork_cult/announce()
+	world << "<b>The game mode is: Clockwork Cult!</b>"
+	world << "<b><span class='brass'>Ratvar</span>, the Clockwork Justiciar, has formed a covenant of Enlightened aboard [station_name()].</b>"
+	world << "<b><span class='brass'>Enlightened</span>: Serve your master so that his influence might grow.</b>"
+	world << "<b><span class='boldannounce'>Crew</span>: Prevent the servants of Ratvar from taking over the station.</b>"
+
+/datum/game_mode/clockwork_cult/pre_setup()
+	if(config.protect_roles_from_antagonist)
+		restricted_jobs += protected_jobs
+	if(config.protect_assistant_from_antagonist)
+		restricted_jobs += "Assistant"
+	var/starter_servants = max(1, round(num_players() / 10)) //Guaranteed one cultist - otherwise, about one cultist for every ten players
+	while(starter_servants)
+		var/datum/mind/servant = pick(antag_candidates)
+		servants_of_ratvar += servant
+		antag_candidates -= servant
+		modePlayer += servant
+		servant.special_role = "Servant of Ratvar"
+		servant.restricted_roles = restricted_jobs
+		starter_servants--
+	return 1
+
+/datum/game_mode/clockwork_cult/post_setup()
+	forge_clock_objectives()
+	for(var/datum/mind/servant in servants_of_ratvar)
```

Comment:
I honestly don't grasp the logic behind that, but Remie is a greater coder than I so I'll oblige you.